Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -12,6 +12,7 @@ namespace DataAccessLayer.Database
     public class DatabaseContext : DbContext
     {
         // TODO: turn into enviornmental variables for dev and deploy
+        readonly string productionDB = "Data Source=pointmapdbinstance.cqugps36mcx5.us-east-2.rds.amazonaws.com;Initial Catalog=pointmapdbinstance;User ID=Administrator;Password=NightWatch_123";

```

Comment:
was this left accidentally?

Reply:
Just having it while i'm still testing this branch to make it easy to deploy, will remove once I'm ready to merge

Reply:
alright